Fixes a bug in scvsmag with preferred origins without arrivals (second try).

FMassin opened the pull request:
Before this, preferred origins that have no arrival would crash scvsmag when using OriginQuality.azimuthalGap.

Now, if it has no arrival, scvsmag exits without doing Mvs, but VS's expirationTime is updated (using the current preferred origin).

This was [issue 92](https://github.com/SeisComP3/seiscomp3/issues/92), this is the second try I closed the first.

gempa-jabe commented:
The author is now "Sysop User" ;)

Basically you don't need to create new pull request. Just rebase your *bug_fix_scvsmag* branch against our master. This is as easy as:

```
$ git rebase origin/master
$ git push -f
```

Now to change the author just amend the last commit with:

```
$ git commit --amend --author "Your Name <email@address.com>"
$ git push -f
```

This will force push your branch and I can pull it.

FMassin commented:
Yes also much cleaner than claiming sysop@eewpb is me 👍

But I also did my changes on the master of SED-EEW fork, providing me to use this fork for other bugs... so I have just added the bug_fix_scvsmag branch and cleaned up my mess in the master, now I need to do the same for the fix of the xml conversion stylesheet !
